src/ray/core_worker/common.h

```diff
@@ -20,6 +20,63 @@ struct RayFunction {
   const std::vector<std::string> function_descriptor;
 };
 
+/// Binary representation of ray object.
+class RayObject {
```

Comment:
better put this `RayObject` class in `src/common/`, because raylet may also need to use this.